packages/sdk/src/restclient.spec.ts

```diff
@@ -188,13 +198,40 @@ describe("RestClient", () => {
   });
 
   describe("authAccounts", () => {
-    it("works", async () => {
+    it("works for unused account without pubkey", async () => {
+      pendingWithoutCosmos();
+      const client = new RestClient(httpUrl);
+      const { result } = await client.authAccounts(unusedAccount.address);
+      expect(result).toEqual({
+        type: "cosmos-sdk/Account",
+        value: {
+          address: unusedAccount.address,
+          public_key: "", // not known to the chain
+          coins: [
+            {
+              amount: "1000000000",
+              denom: "ucosm",
+            },
+            {
+              amount: "1000000000",
+              denom: "ustake",
+            },
+          ],
+          account_number: 5,
+          sequence: 0,
+        },
+      });
+    });
+
+    it("has correct pubkey for faucet", async () => {
       pendingWithoutCosmos();
       const client = new RestClient(httpUrl);
-      const { result } = await client.authAccounts(faucetAddress);
-      const account = result.value;
-      expect(account.account_number).toEqual(4);
-      expect(account.sequence).toBeGreaterThanOrEqual(0);
+      const { result } = await client.authAccounts(faucet.address);
+      expect(result.value).toEqual(
+        jasmine.objectContaining({
+          public_key: encodeBech32Pubkey(faucet.pubkey, "cosmospub"),
+        }),
+      );
```

Comment:
Those two tests are the hear of this PR

Reply:
As above, I would adjust authAccounts to do the decode step, so we can compare directly with faucet.pubkey